docs/runbook/Quick_Start.md

```diff
@@ -59,8 +59,8 @@ python3 -m unittest discover terragraph_planner -b
 
 ### Configuration File
 
-To customize and run a plan, you will need an input configuration yaml file.
-Refer to [template.yaml](https://github.com/terragraph/terragraph-planner/blob/main/terragraph_planner/data/template.yaml)
+One way customize and run a plan uses an input configuration yaml file. Refer to
```

Comment:
I wonder if it looks better.
"One way customize and run a plan uses an input configuration yaml file" -> "One way to customize and run a plan is using an input configuration yaml file".